java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -91,6 +94,13 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
         baseTransaction.configResetStat();
 
+        baseTransaction
+            .pfadd(hllKey1, new String[0])
+            .pfadd(hllKey1, new String[] {"a", "b", "c"})
+            .pfadd(hllKey2, new String[] {"b", "c", "d"})
+            .pfadd(hllKey1, new String[0]);
```

Comment:
what's the benefit of testing all of this?

Reply:
To satisfy the god of the tests